src/Controls/src/Core/Compatibility/Handlers/Shell/iOS/ShellSectionRenderer.cs

```diff
@@ -775,8 +774,6 @@ public override void WillShowViewController(UINavigationController navigationCon
 						navBarVisible = Shell.GetNavBarIsVisible(element);
 				}
 
-				navigationController.SetNavigationBarHidden(!navBarVisible, true);
```

Comment:
Are we sure this doesn't break anything? What's the advantage of removing this line?